.github/workflows/ci-post-land.yml

```diff
@@ -119,7 +147,7 @@ jobs:
   build_ci_base_docker_image:
     needs: prepare
     runs-on: ubuntu-latest-xl
-    if: ${{ needs.prepare.outputs.base-image-changes == 'true' && github.event_name == 'push' }}
+    if: ${{ needs.prepare.outputs.base-image-changes == 'true' || github.event_name == ‘create’ }}
```

Comment:
I cannot tell for sure, but the single quotes look a bit different, ie ` vs '.

Reply:
changed the ticks, also forced to rebase.